src/libraries/System.Configuration.ConfigurationManager/tests/System/Configuration/BinaryFormatterDeprecationTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Configuration;
+using Xunit;
+
+namespace System.ConfigurationTests
+{
+    public class BinaryFormatterDeprecationTests
+    {
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "SettingsSerializeAs.Binary is deprecated only on Core")]
+        [Fact]
+        public void ThrowOnSettingsPropertyConstructorWithSettingsSerializeAsBinary()
+        {
+#pragma warning disable CS0618 // Type or member is obsolete
+            Assert.Throws<NotSupportedException>(() => new SettingsProperty("Binary", typeof(byte[]), null, false,"AString", SettingsSerializeAs.Binary, new SettingsAttributeDictionary(), true, true));
+#pragma warning restore CS0618 // Type or member is obsolete
+        }
+
+        [Fact]
+        public void SerializeAndDeserializeWithSettingsSerializeAsBinary()
+        {
+            AppContext.SetSwitch("System.ComponentModel.TypeConverter.EnableUnsafeBinaryFormatterInPropertyValueSerialization", true);
```

Comment:
Usually tests that make changes on AppContext are done via RemoteExecutor so as to not potentially affect other tests. Would you mind transforming both of these tests to use RemoteExecutor instead?